cassovary-core/src/main/scala/com/twitter/cassovary/util/FastUtilUtils.scala

```diff
@@ -61,4 +61,6 @@ object FastUtilUtils {
 
   def newInt2IntOpenHashMap(): mutable.Map[Int, Int] =
```

Comment:
I agree that's easier to use IndexedSeq, but I'm slightly afraid that if I used map, etc. on IndexedSeq I'll accidentally do an O(n) copy when I'm not expecting it.